test/integration_test.rb

```diff
@@ -29,7 +29,7 @@ def test_ruby_lsp_check_works
   def test_activation_script_succeeds_even_on_binary_encoding
     ENV["LC_ALL"] = "C"
     ENV["LANG"] = "C"
-    ENV["PS1"] = "\xE2".b
+    ENV["PS1"] = "\xE2\x96\xB7".b
```

Comment:
We still want to verify that we're rejecting values that cannot be converted to UTF-8. Can you please duplicate this test so that we have examples for both?

Reply:
Done. I'm not sure how the result of this is used by the extension – do we also want this test to assert that valid unicode in ENV is successfully returned from the activation script?

Reply:
I tried creating an end to end test using the extension, but there was no easy way to get the `LANG=C` to apply to the shell invoked by NodeJS when running our activation script.

For now, it's fine to just keep these integration tests.